utilities/shared.mjs

```diff
@@ -19,20 +21,25 @@ export function isValidJSON(input=""){
  * @param input A string which should be a valid Integer number
  * @return boolean For whether or not the supplied string was a valid Integer number
  */ 
-export function validateID(id){
-   if(!isNaN(id)){
+export function validateID(id, type="mongo"){
```

Comment:
We will eventually remove this entirely from the utils, since the db drivers will be the authorities on what's a valid id.